cert-data.yml

```diff
@@ -1,4 +1,179 @@
 ---
+### 4_programmatic Default course
+Default/OpenEdX/Certificate: &cert_defaults
+  VERSION: 4_programmatic
+  TEMPLATEFILE: False
+  instructors:
+    - instructor: &instructor_1
+        x_position: 475
+        y_position: 140
+    - instructor: &instructor_2
+        x_position: 475
+        y_position: 220
+    - instructor: &instructor_3
+        x_position: 475
+        y_position: 300
+    - instructor: &instructor_4
+        x_position: 475
+        y_position: 380
+    - instructor: &instructor_5
+        x_position: 275
+        y_position: 140
+    - instructor: &instructor_6
+        x_position: 75
+        y_position: 140
+  instructor_template:
+    # ordered list of steps to render the instructor signature block, bottom to top
+    - text:
+        string: "school"
+        font_size: 10
+        leading: 10
+        text_color: [.18,.18,.18]
+        alignment: TA_RIGHT
+        width: 250
+        height: 12
+    - text:
+        string: "title_2"
+        font_size: 10
+        leading: 10
+        text_color: [.18,.18,.18]
+        alignment: TA_RIGHT
+        width: 250
+        height: 11
+    - text:
+        string: "title"
+        font_size: 10
+        leading: 10
+        text_color: [.18,.18,.18]
+        alignment: TA_RIGHT
+        width: 250
+        height: 11
+    - text:
+        string: "name"
+        font_size: 12
+        leading: 12
+        text_color: [.18,.18,.18]
+        alignment: TA_RIGHT
+        width: 250
+        height: 12
+    - image:
+        file: "signature_file"
+        width: 200
+        height: 50
+        x_offset: 50
+  render_template:
+    # ordered list of steps to render the template, back to front
+    - rectangle:
+        x: 81
+        y: 54
+        width: 680
+        height: 488
+        stroke: 1
+        fill: 1
+        stroke_width: 2
+        stroke_color: [.51,.15,.22]
+        fill_color: [.87,.85,.88]
+    - rectangle:
+        x: 85
+        y: 58
+        width: 672
+        height: 480
+        stroke: 1
+        fill: 1
+        stroke_width: 2
+        stroke_color: [.71,.64,.72]
+        fill_color: [1,1,1]
+    - image:
+        x: 120
+        y: 485
+        width: 150
+        height: 30
+        file: "images/openedx.png"
+    - text:
+        string: "<b>CERTIFICATE OF ACHIEVEMENT</b>"
+        font_size: 9.9
+        leading: 9.9
+        text_color: [.51,.15,.22]
+        alignment: TA_LEFT
+        x: 121
+        y: 474
+        width: 150
+        height: 12
+  course_template:
+    # ordered list of steps to render the template, top to bottom
+    - text:
+        string: "{date_string}"
+        font_size: 12
+        leading: 14
+        text_color: [.13,.14,.22]
+        alignment: TA_RIGHT
+        x: 120
+        y: 488
+        width: 600
+        height: 14
+    - text:
+        string: "<b>{student_name}</b>"
```

Comment:
We need to document all possible string formats.

Can you add these to the README?

We should probably have an entire section devoted to V4.